sakura_core/plugin/CPlugin.cpp

```diff
@@ -262,7 +265,7 @@ std::vector<std::wstring> wstring_split( std::wstring sTrg, wchar_t cSep )
         splitVec.push_back( sTrg.substr( 0, idx ) );
         sTrg = sTrg.substr( ++idx );
     }
-	if (sTrg != L"") {
+	if (!sTrg.empty()) {
```

Comment:
```suggestion
	if (sTrg.length()) {
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17